tool/e2e-tests/test-packs/test-packs.ts

```diff
@@ -4,15 +4,24 @@ import path from "path"
 import {sendPageTitle} from "../locators"
 
 export async function loadClinicalTestPack1Examples(driver: ThenableWebDriver): Promise<void> {
-  const testPackUpload = await getTestPackUpload(driver)
-  uploadTestPack(testPackUpload, "Full Prescriber Volume Pack.xlsx")
-  await loadPrescriptionsFromTestPack(driver)
-  await driver.wait(until.elementsLocated(sendPageTitle), apiTimeout)
+  await loadTestPack(driver, "Full Prescriber Volume Pack.xlsx")
 }
 
 export async function loadSupplierTestPack1Examples(driver: ThenableWebDriver): Promise<void> {
+  await loadTestPack(driver, "Supplier 1 Test Pack.xlsx")
+}
+
+export async function loadPrescriptionTypeTestPack1Examples(driver: ThenableWebDriver): Promise<void> {
+  await loadTestPack(driver, "Prescription Types Test Pack.xlsx")
+}
+
+export async function loadPrescriptionTypeTestPack2Examples(driver: ThenableWebDriver): Promise<void> {
```

Comment:
is it worth naming these a bit more specifically? rather than TP1/TP2, should we match the name of the file (so for example `loadBadScriptTypesTestPack` or something)